data/json/npcs/isolated_road/isolated_road_cody_prototype.json

```diff
@@ -356,10 +356,10 @@
     "id": "TALK_BLACKSMITH_LOUVERTURE",
     "//": "Sun Mace",
     "type": "talk_topic",
-    "dynamic_line": "Oh that's one of my sillier ideas.  Jay is convinced it will never work but I really want to give it a shot.  A traditional mace with gas venting ports and an ignition system for mini bursts of fire.  Maybe I've just played too many fantasy RPGs but it seems worth a shot.  To get this one working I'm gonna need a real deal mace, something that can handle the pressure and additional parts as well as a military grade flamethrower.  Fabrication will definitely take 2 weeks on this one.",
+    "dynamic_line": "Oh, that's one of my sillier ideas.  Jay is convinced it'll never work but I really want to give it a shot.  A traditional mace with gas-venting ports and an ignition system for mini bursts of fire.  Maybe I've just played too many RPGs, but it seems worth a shot.  To get this one working I'm gonna need a real deal mace, something - that can handle the pressure - as well as a military grade flamethrower.  Fabrication will definitely take two weeks on this one.",
```

Comment:
I don't understand the grammatical significance of these "-" throughout the lines. Though I think it's just ignorance on my part.

Reply:
`gas-venting` is used to describe the ports - the ports vent gas, so they're called gas-venting. The bit about handling the pressure is actually slightly miswritten, so imma fix that.

Reply:
yeah the "- that can handle the pressure -" is what I meant